### PR TITLE
EX-17: Allows imports with numeric identifier

### PR DESCRIPTION
**Pull request by damien-carcel**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.4.1 (2015-10-13)
+ - Allow init import and product import with numeric identifier
```
Comment (rybus) [inline CHANGELOG.md]: [...] having numeric identifier

File: Reader/AbstractIteratorReader.php [hunk 2]
```diff
@@ -63,7 +63,28 @@ public function read()
...
+
+    /**
+     * Converts an entity numerical identifier ('sku' for products,
+     * 'code' for other entities) into string to allow import.
+     *
+     * @param array $item
+     *
+     * @return array
+     */
+    protected function convertNumericIdentifierToString(array $item)
```
Comment (rybus) [inline Reader/AbstractIteratorReader.php]: This method should be abstract

File: Reader/FileIteratorReader.php [hunk 3]
```diff
@@ -175,11 +176,23 @@ protected function createIterator()
 
     /**
      * Returns the options for the iterator
```
Comment (rybus) [inline Reader/FileIteratorReader.php]: Returns options of the iterator

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 1.4.1 (2015-10-13)
```
Comment (rybus) [inline CHANGELOG.md]: You should not set the release date in this commit. A dedicated commit updating CHANGELOG.md and README.md will have to be pushed to set this information ;)

File: Reader/AbstractIteratorReader.php (hunk 2 above)
Comment (damien-carcel) [inline Reader/AbstractIteratorReader.php]: No, because it will impose to implement it everywhere, which is a BC-break.

Comment (rybus) [inline Reader/AbstractIteratorReader.php]: Ok

**Comment by rybus** [comment]
GTM

**Pull request merged**